404.html

```diff
@@ -29,62 +29,244 @@
     <meta name="viewport" content="width=device-width" />
     <title>Brackets - Page Not Found (404)</title>
 
-    <link rel="stylesheet" href="/css/normalize.css" />
-    <link rel="stylesheet" href="/css/foundation.min.css" />
+    <link rel="stylesheet" href="./css/normalize.css" />
+    <link rel="stylesheet" href="./css/foundation.min.css" />
+    <link rel="stylesheet" href="./css/brackets.min.css" />
 
-    <link rel="stylesheet" href="/css/brackets.io.css">
-    
+    <!-- Linked Favicon -->
+    <link rel="shortcut icon" href="favicon.ico" />
```

Comment:
Has the icon and the fonts below at lines 39 and 40 legally validated to be used in the site? Check with PgM once on the legal.